### PR TITLE
[script.video.bgtime_tv] 1.0.3

### PR DESCRIPTION
**Pull request by portokala123**
### Description
Plugin for our service https://bgtime.tv/. You need to have active account to use it.

### Checklist:
- [x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

### CODE REVIEW

**Comment by portokala123** [comment]
Changes are implemented in the new commit.

**Comment by romanvm** [comment]
LGTM

File: script.video.bgtime_tv/addon.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+			This addon provides access to streams of BGTime.TV (live channels and series).
+			It requires active BGTime.TV registration.
+		</description>
+		<disclaimer lang="bg_BG">
+			Нужни са активни потребителско име и парола.
+			Може да се ползва само извън България. За детайли виж https://bgtime.tv/countries
+		</disclaimer>
+		<website>https://bgtime.tv/</website>
+		<email>support@mytv.bg</email>
+		<news>v1.0.3 created by mariapmarkova@gmail.com</news>
```
Comment (enen92) [inline script.video.bgtime_tv/addon.xml]: Repeated tag here. Not sure this can be localized.

Comment (MartijnKaijser) [inline script.video.bgtime_tv/addon.xml]: news tag cannot be localised so only English should be kept

Comment (portokala123) [inline script.video.bgtime_tv/addon.xml]: Fixed.

**Pull request merged**